ads/google/a4a/utils.js

```diff
@@ -144,24 +144,17 @@ export function googleBlockParameters(a4a, opt_experimentIds) {
   win['ampAdGoogleIfiCounter'] = win['ampAdGoogleIfiCounter'] || 1;
   const slotRect = a4a.getPageLayoutBox();
   const iframeDepth = iframeNestingDepth(win);
-  // Detect container types.
-  const containerTypeSet = {};
-  for (let el = adElement.parentElement, counter = 0;
-      el && counter < 20; el = el.parentElement, counter++) {
-    const tagName = el.tagName.toUpperCase();
-    if (ValidAdContainerTypes[tagName]) {
-      containerTypeSet[ValidAdContainerTypes[tagName]] = true;
-    }
-  }
+  const enclosingContainers = getEnclosingContainerTypes(adElement);
   const pfx =
-      (containerTypeSet[ValidAdContainerTypes['AMP-FX-FLYING-CARPET']]
-       || containerTypeSet[ValidAdContainerTypes['AMP-STICKY-AD']])
+      (enclosingContainers.indexOf(
+          ValidAdContainerTypes['AMP-FX-FLYING-CARPET']) != -1
```

Comment:
can you use includes:  

const pfx = enclosingContainers.includes(ValidAdContainerTypes['AMP-FX-FLYING-CARPET']) || enclosingContainers.includes(ValidAdContainerTypes['AMP-STICKY-AD']);